### DISCUSSION TITLE
How to set custom workspace as a default one

### DISCUSSION THREAD

**Discussion by NeverL4nd2** [nikitabobko/AeroSpace#619] [questions-and-answers] [answered]
In my config I'm using Meh (alt-shift-ctrl) + qwerty as a way to move to the named workspace, I think the 6 workspaces are enough for me and it's ergonomic for me.
The problem I encountered is that by default the '1' workspace is launched, who to set it to "worksplace q"?
and also remove rest of the workspaces from tray

**Comment by tylerdak** [comment, marked as answer]
Use an after-startup hook to run `workspace q`

Edit: Correction made to hook name

**Comment by AlexBurdu** [comment]
Is there any example on how to configure an `after-setup ` hook? I can't find any reference in the documentation.

**Reply by tylerdak** [reply]
I think I got the hook name wrong. Looks like after-startup-command is the config key, and it accepts multiple commands like a shortcut would.

https://nikitabobko.github.io/AeroSpace/guide#:~:text=after%2Dstartup%2Dcommand%20%3D%20%5B%5D
